client/lib/two-step-authorization/index.js

```diff
@@ -234,6 +300,14 @@ TwoStepAuthorization.prototype.isTwoStepSMSEnabled = function() {
 	return this.data ? this.data.two_step_sms_enabled : false;
 };
 
+TwoStepAuthorization.prototype.isSecurityKeyEnabled = function() {
+	return this.data?.two_step_webauthn_enabled ?? false;
```

Comment:
seeing the `data?` here made me realize that `data` can be undefined / null and made me question all the direct setting above that @tyxla also pointed out, which made me have to go through every flow and revalidate it. 

Can we alter the code so that `this.data` is always defined? That would save future folks from having to do all the mental gymnastics to validate that the code is correct.

Reply:
Thanks, this was fixed in 89714c6